CRITICAL - 2021-09-11 10:22:12 --> Undefined variable: erros_model
#0 C:\xampp\htdocs\ordem\app\Views\Layout\_mensagens.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 46, Array)
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(487): CodeIgniter\View\View->render('Layout/_mensage...', NULL, true)
#4 C:\xampp\htdocs\ordem\app\Views\Layout\principal.php(170): CodeIgniter\View\View->include('Layout/_mensage...')
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(246): CodeIgniter\View\View->render('Layout/principa...', Array, true)
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('Ordens/itens', Array, true)
#9 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(35): view('Ordens/itens', Array)
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->itens('RLAJBT2IANSMCJE...')
#11 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#12 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2021-09-11 10:55:26 --> Unknown column 'id_principal' in 'where clause'
CRITICAL - 2021-09-11 10:55:26 --> Unknown column 'id_principal' in 'where clause'
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Model.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(644): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(340): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(256): App\Controllers\OrdensItens->buscaOrdemItemOu404(2, 1009)
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->atualizarQuantidade('RLAJBT2IANSMCJE...')
#9 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2021-09-11 11:04:30 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\ordem\app\Views\Layout\_mensagens.php(46): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 46, Array)
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(487): CodeIgniter\View\View->render('Layout/_mensage...', NULL, true)
#4 C:\xampp\htdocs\ordem\app\Views\Layout\principal.php(170): CodeIgniter\View\View->include('Layout/_mensage...')
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(246): CodeIgniter\View\View->render('Layout/principa...', Array, true)
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('Ordens/itens', Array, true)
#9 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(35): view('Ordens/itens', Array)
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->itens('RLAJBT2IANSMCJE...')
#11 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#12 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2021-09-11 11:20:35 --> Não há dados para update.
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(1690): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(880): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Model.php(718): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Ordem))
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(683): CodeIgniter\Model->update('1009', Object(App\Entities\Ordem))
#4 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(277): CodeIgniter\BaseModel->save(Object(App\Entities\Ordem))
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->atualizarQuantidade('RLAJBT2IANSMCJE...')
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-09-11 11:22:13 --> Não há dados para update.
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(1690): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(880): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Model.php(718): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Ordem))
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(683): CodeIgniter\Model->update('1009', Object(App\Entities\Ordem))
#4 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(278): CodeIgniter\BaseModel->save(Object(App\Entities\Ordem))
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->atualizarQuantidade('RLAJBT2IANSMCJE...')
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-09-11 11:22:36 --> Não há dados para update.
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(1690): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(880): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Model.php(718): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Ordem))
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(683): CodeIgniter\Model->update('1009', Object(App\Entities\Ordem))
#4 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(278): CodeIgniter\BaseModel->save(Object(App\Entities\Ordem))
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->atualizarQuantidade('RLAJBT2IANSMCJE...')
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-09-11 11:50:42 --> Não há dados para update.
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(896): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Model.php(718): CodeIgniter\BaseModel->update(NULL, Array)
#2 C:\xampp\htdocs\ordem\app\Models\OrdemItemModel.php(50): CodeIgniter\Model->update()
#3 C:\xampp\htdocs\ordem\app\Controllers\OrdensItens.php(278): App\Models\OrdemItemModel->atualizarQuantidadeItem(Object(stdClass))
#4 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\OrdensItens->atualizarQuantidade('RLAJBT2IANSMCJE...')
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrdensItens))
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-09-11 13:10:08 --> syntax error, unexpected '}', expecting ';'
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(273): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(247): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Usua...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Usua...')
#3 C:\xampp\htdocs\ordem\app\Libraries\Autenticacao.php(14): spl_autoload_call('App\\Models\\Usua...')
#4 C:\xampp\htdocs\ordem\app\Config\Services.php(29): App\Libraries\Autenticacao->__construct()
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Config\BaseService.php(192): Config\Services::autenticacao(false)
#6 C:\xampp\htdocs\ordem\app\Config\Services.php(26): CodeIgniter\Config\BaseService::getSharedInstance('autenticacao')
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Common.php(1084): Config\Services::autenticacao()
#8 C:\xampp\htdocs\ordem\app\Filters\LoginFilter.php(28): service('autenticacao')
#9 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Filters\Filters.php(181): App\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), NULL)
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(407): CodeIgniter\Filters\Filters->run('grupos', 'before')
#11 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-09-11 13:14:59 --> Unknown column 'grupos_usuarios' in 'on clause'
CRITICAL - 2021-09-11 13:14:59 --> Unknown column 'grupos_usuarios' in 'on clause'
#0 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('SELECT `usuario...')
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `usuario...')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `usuario...')
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1881): CodeIgniter\Database\BaseConnection->query('SELECT `usuario...', Array, false)
#4 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Model.php(197): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\BaseModel.php(603): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ordem\app\Models\UsuarioModel.php(202): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(272): App\Models\UsuarioModel->recuperaResponsaveisParaOrdem()
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Ordens->recuperaResponsaveis()
#9 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ordens))
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2021-09-11 15:48:16 --> Trying to get property 'email' of non-object
#0 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(469): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 469, Array)
#1 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(368): App\Controllers\Ordens->enviaOrdemEmAndamentoParaCliente(Object(App\Entities\Ordem))
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Ordens->email('RLAJBT2IANSMCJE...')
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ordens))
#4 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2021-09-11 15:52:13 --> Trying to get property 'nome' of non-object
#0 C:\xampp\htdocs\ordem\app\Views\Ordens\ordem_andamento_email.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('Ordens/ordem_an...', Array, true)
#4 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(485): view('Ordens/ordem_an...', Array)
#5 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(368): App\Controllers\Ordens->enviaOrdemEmAndamentoParaCliente(Object(App\Entities\Ordem))
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Ordens->email('RLAJBT2IANSMCJE...')
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ordens))
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2021-09-11 15:56:14 --> Undefined variable: valorProduto
#0 C:\xampp\htdocs\ordem\app\Views\Ordens\ordem_andamento_email.php(52): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('Ordens/ordem_an...', Array, true)
#4 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(485): view('Ordens/ordem_an...', Array)
#5 C:\xampp\htdocs\ordem\app\Controllers\Ordens.php(368): App\Controllers\Ordens->enviaOrdemEmAndamentoParaCliente(Object(App\Entities\Ordem))
#6 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Ordens->email('RLAJBT2IANSMCJE...')
#7 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ordens))
#8 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ordem\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ordem\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
